.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+### Description of change
+
+_Please provide a description of change here._
+
+### Pull Request check-list
+
+_Please make sure to review and check all of these items:_
+
+- [ ] Does `make -j8 test` (or `vcbuild test nosign`) pass with this change
```

Comment:
..and possibly go with `make -j4` since laptops or lower end machines could run into parallelism issues.